pkg/controller/clusterroleaggregation/clusterroleaggregation_controller.go

```diff
@@ -49,8 +49,8 @@ type ClusterRoleAggregationController struct {
 }
 
 // NewClusterRoleAggregation creates a new controller
-func NewClusterRoleAggregation(clusterRoleInformer rbacinformers.ClusterRoleInformer, clusterRoleClient rbacclient.ClusterRolesGetter) *ClusterRoleAggregationController {
-	c := &ClusterRoleAggregationController{
+func NewClusterRoleAggregation(clusterRoleInformer rbacinformers.ClusterRoleInformer, clusterRoleClient rbacclient.ClusterRolesGetter) *Controller {
```

Comment:
Should consider making this NewController for consistency.

Reply:
Hmm, sounds good. Will do the changes.

Reply:
@cheftako Done.